builder/bundleset.go

```diff
@@ -32,8 +32,10 @@ type bundle struct {
 	Header   bundleHeader
 
 	DirectIncludes []string
-	DirectPackages []string
+	DirectPackages map[string]bool
 	AllPackages    []string
+
+	Files map[string]bool
```

Comment:
Is this actually used? I'm probably just missing it, but all references to `Files` I see in this patch are to the `bundleinfo` type or the `Manifest` type.

Reply:
Uh, yes. This is probably the most important data structure of this entire PR. Look in chroots.go. This is where all the files are stored (instead of there being actual chroots). This exact struct then gets written to a JSON file.

Reply:
:wink: 'twas a case of GitHub helpfully hiding the entire `chroots.go` file